File: PIPromptClases/MyApplication4/app/src/main/java/com/example/myapplication/UsuarioComunidad.kt
package com.example.myapplication

data class UsuarioComunidad(
    var usuarioComunidadId: Int,
    var usuarioId: Int,
    var comunidadId: Int,
    var rol: Rol
) {
    constructor() : this(
        usuarioComunidadId = 0,
        usuarioId = 0,
        comunidadId = 0,
        rol = Rol()
    )
}

File: frontend-wpf/C#Clases/UsuarioComunidad.cs
public class UsuarioComunidad
    {
        public int UsuarioComunidadId { get; set; }
        public int UsuarioId { get; set; }
        public int ComunidadId { get; set; }
        public Rol Rol { get; set; }

        public UsuarioComunidad() { }

        public UsuarioComunidad(int usuarioComunidadId, int usuarioId, int comunidadId, Rol rol)
        {
            UsuarioComunidadId = usuarioComunidadId;
            UsuarioId = usuarioId;
            ComunidadId = comunidadId;
            Rol = rol;
        }
    }